tensorstore/kvstore/tsgrpc/tsgrpc.cc

```diff
@@ -137,7 +138,11 @@ struct TsGrpcKeyValueStoreSpecData {
       jb::Member(
           DataCopyConcurrencyResource::id,
           jb::Projection<
-              &TsGrpcKeyValueStoreSpecData::data_copy_concurrency>()) /**/
+              &TsGrpcKeyValueStoreSpecData::data_copy_concurrency>()),
+      jb::Member("max_sent_part_bytes",
```

Comment:
I think that we just use something like this:

constexpr size_t kMaxPayloadSize = 2 * 1024 * 1024;  // 2mb, gRPC allows 4mb.

Reply:
... which means that there should be a separate test which uses a 4mb buffer, or so.